src/test/java/uk/gov/companieshouse/api/accounts/service/impl/FilingServiceImplTest.java

```diff
@@ -0,0 +1,468 @@
+        SimpleDateFormat formatter = new SimpleDateFormat(TRANSACTION_DATE_FORMAT);
+        Date date = formatter.parse(dateInString);
+
+        return date;
+    }
+
+    /**
+     * Builds the small full accounts model. Functionality needs to be change.
+     */
+    private Account getSmallFullAccount() {
+        Account account = new Account();
+        account.setPeriod(getAccountPeriod());
+        account.setBalanceSheet(getBalanceSheet());
+        account.setNotes(getNotes());
+        account.setCompany(getCompany());
+
+        return account;
+    }
+
+    private Company getCompany() {
+        Company company = new Company();
+        company.setCompanyName(COMPANY_NAME);
+        company.setCompanyNumber(COMPANY_NAME);
+
+        return company;
+    }
+
+    private Notes getNotes() {
+        Notes notes = new Notes();
+
+        PostBalanceSheetEvents postBalanceSheetEvents = new PostBalanceSheetEvents();
+        postBalanceSheetEvents.setCurrentPeriodDateFormatted(CURRENT_PERIOD_FORMATTED);
+        postBalanceSheetEvents.setPostBalanceSheetEventsInfo("test post balance note");
+
+        notes.setPostBalanceSheetEvents(postBalanceSheetEvents);
+
+        return notes;
+    }
+
+    private Period getAccountPeriod() {
+        Period period = new Period();
+        period.setCurrentPeriodStartOn(CURRENT_PERIOD_START_ON);
+        period.setCurrentPeriodEndsOn(CURRENT_PERIOD_END_ON);
+        period.setPreviousPeriodStartOn(PREVIOUS_START_ON);
+        period.setPreviousPeriodEndsOn(PREVIOUS_PERIOD_END_ON);
+
+        return period;
+    }
+
+    /**
+     * Build BalanceSheet model by using information from database
+     */
+    private BalanceSheet getBalanceSheet() {
+        BalanceSheet balanceSheet = new BalanceSheet();
+
+        CalledUpSharedCapitalNotPaid calledUpSharedCapitalNotPaid = new CalledUpSharedCapitalNotPaid();
+        calledUpSharedCapitalNotPaid.setCurrentAmount(9);
+        calledUpSharedCapitalNotPaid.setPreviousAmount(99);
+
+        balanceSheet.setCalledUpSharedCapitalNotPaid(calledUpSharedCapitalNotPaid);
+        balanceSheet.setCurrentPeriodDateFormatted(CURRENT_PERIOD_FORMATTED);
+        balanceSheet.setPreviousPeriodDateFormatted(PREVIOUS_PERIOD_FORMATTED);
+
+        return balanceSheet;
+    }
+
+    /**
+     * Get file content.
+     *
+     * @param filePathName - contains file location and name.
+     * @return
+     * @throws URISyntaxException
+     * @throws IOException
+     */
+    private String getFileContentFromResource(String filePathName)
+        throws URISyntaxException, IOException {
+
+        Path path = Paths.get(getClass().getClassLoader()
+            .getResource(filePathName).toURI());
+
+        StringBuilder data = new StringBuilder();
+        Stream<String> lines = Files.lines(path);
+        lines.forEach(line -> data.append(line).append("\n"));
+        lines.close();
+
+        return data.toString();
```

Comment:
This can be simplified to:
```java
private String getFileContentFromResource(String filePathName) throws URISyntaxException, IOException {
    	Path path = Paths.get(getClass().getClassLoader().getResource(filePathName).toURI());
    	return new String(Files.readAllBytes(path));
}
````

Reply:
💯 Good shout!!!!

Reply:
9f407eb